packages/mask/src/plugins/Web3Feed/SNSAdaptor/ReversedAddress.tsx

```diff
@@ -0,0 +1,26 @@
+import { memo } from 'react'
+import type { NetworkPluginID } from '@masknet/web3-shared-base'
+import { useReverseAddress, useWeb3State } from '@masknet/plugin-infra/web3'
+import { ZERO_ADDRESS } from '@masknet/web3-shared-evm'
+
+interface ReverseAddressProps {
+    address?: string
+    pluginId?: NetworkPluginID
+    domainSize?: number
+    size?: number
+    fontSize?: string
+    fontWeight?: number
+}
+
+export const ReversedAddress = memo<ReverseAddressProps>(
```

Comment:
Why not use `<ReversedAddress />` from `@masknet/shared`?

Reply:
because I need reversed address contained by <span> instead of <Typography> for not folding